hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerTokenTest.java

```diff
@@ -1300,6 +1333,17 @@ void tokenMint() {
         assertTokenTransferInRepository(TOKEN_ID, PAYER, CREATE_TIMESTAMP, INITIAL_SUPPLY);
         assertTokenTransferInRepository(TOKEN_ID, PAYER, mintTimestamp, amount);
         assertTokenInRepository(TOKEN_ID, true, CREATE_TIMESTAMP, mintTimestamp, SYMBOL, INITIAL_SUPPLY + amount);
+
+        var listAssert = assertThat(contractLogRepository.findAll())
+                .hasSize(2);
```

Comment:
To simplify and make it clear what we're testing, we should only lookup and verify the mint contract log. You can verify the overall counts of logs, but should just verify fields for the transaction under test. Same comment for other tests.
```suggestion
        assertThat(contractLogRepository.findById(...))
            .returns(foo, ContractLog::bar)
            ...
```